x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/package_policy_input_var_field.tsx

```diff
@@ -17,12 +17,17 @@ import {
   EuiFieldPassword,
   EuiCodeBlock,
 } from '@elastic/eui';
+import styled from 'styled-components';
 
 import type { RegistryVarsEntry } from '../../../../types';
 import { CodeEditor } from '../../../../../../../../../../src/plugins/kibana_react/public';
 
 import { MultiTextInput } from './multi_text_input';
 
+const FixedWidthDiv = styled.div`
```

Comment:
Do I misunderstand this or the variable should be named `FixedHeightDiv`?

Reply:
You're totally right! fixing it